lib/tail-until.js

```diff
@@ -16,18 +16,24 @@ async function tailUntil (filePath, until, timeout = 5000) {
     return stdout.indexOf(until) > -1;
   };
 
-  return new B(async (resolve, reject) => {
+  return await new B((resolve, reject) => {
     let started = proc.start(startDetector);
 
     /* eslint-disable promise/prefer-await-to-then */
     let timedout = B.delay(timeout).then(() => {
-      return reject(`tailing file ${filePath} failed after ${timeout}ms`);
+      return reject(new Error(`Tailing file ${filePath} failed after ${timeout}ms`));
     });
     /* eslint-enable */
 
-    await B.race([started, timedout]);
+    return (async () => {
```

Comment:
since we're already in a promise context here, I wonder if it's more readable to just do:

```js
B.race([started, timedout]).then(resolve).catch(reject);
```